qiskit/tools/visualization.py

```diff
@@ -661,20 +656,19 @@ def plot_wigner_data(wigner_data, phis=None,
                 point: a single point in phase space
                 curve: a two dimensional curve
                 plaquette: points plotted as circles
-        
+
     Returns:
         none: plot is shown with matplotlib to the screen
     """
-    
-    if method is None:
-        wigDim = len(np.shape(wigner_data))
-        if wigDim == 1:
+    if not method:
+        wig_dim = len(np.shape(wigner_data))
+        if wig_dim == 1:
             if np.shape(wigner_data) == 1:
                 method = 'point'
             else:
                 method = 'curve'
-        elif wigDim ==2:
-            method ='plaquette'
+        elif wig_dim == 2:
+            method = 'plaquette'
 
     if method == 'curve':
         plot_wigner_curve(wigner_data, xaxis=phis)
```

Comment:
In the next lines:

```
     elif method == 'state':
         wigner_function(wigner_data, text_out)
```

`wigner_function()` seems not to be a valid function name: can you double-check?

Reply:
we fixed the function name

@diego-plan9 did we get everything please?